### PR TITLE
done buttons component

### PR DESCRIPTION
**Pull request by srishtigautam21**

### CODE REVIEW

File: component/Buttons/button.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
+.button{
+    background-color:var(--primary-color);
+    color:var(--white-color);
+    border-radius:3px;
+    border:0px solid var(--white-color);
+    font-size: 1.8rem;
+    font-weight: 400;
+    margin:2rem 2rem;
+    width:12vw;
```
Comment (iRohitGaur) [inline component/Buttons/button.css]: Don’t use vw and vh here. Try checking this in mobile. The button will get very small.

File: css/components.css [hunk 2]
```diff
@@ -4,19 +4,24 @@
...
 
 :root{
     --primary-color:#22c55e;
     --secondary-color:#bbf7d0;
+    --primary-hover:#16a34a;
     --notification-color:#f97316;
     --offgreen:#f0fdf4;
     --darkerofseconadary:#86efac;
-    
+    --box-shadow:gray;
```
Comment (iRohitGaur) [inline css/components.css]: convert this to hex for consistency

File: component/Buttons/button.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+.link-button:hover{
+    background-color: var(--link-hover);
+    cursor:pointer;
+}
+.text-link{
+    text-decoration: underline;
+    font-size: 2.5rem;
+    font-weight: 400;
+    color:var(--black);
+    border:0px solid var(--white-color);
```
Comment (rutu14) [inline component/Buttons/button.css]: You can apply border as none, if you don't want the button border.

**Review by iRohitGaur — APPROVED** [review]

File: component/Buttons/button.css (hunk 1 above)
Comment (srishtigautam21) [reply component/Buttons/button.css]: should i use rem in place of them.

File: css/components.css (hunk 2 above)
Comment (srishtigautam21) [reply css/components.css]: done

File: component/Buttons/button.css (hunk 3 above)
Comment (srishtigautam21) [reply component/Buttons/button.css]: ok done

File: component/Buttons/button.css (hunk 1 above)
Comment (srishtigautam21) [reply component/Buttons/button.css]: done

Comment (srishtigautam21) [reply component/Buttons/button.css]: done

File: component/Buttons/button.css (hunk 3 above)
Comment (srishtigautam21) [reply component/Buttons/button.css]: done

**Comment by humzahasan** [comment]
Looks good. For svg, I would advise you to have svg files seperate and use it as img with src of svg. 

```
<img src="demo.svg" />
```

**Review by humzahasan — APPROVED** [review]
Looks good. For further enhancement have svg seperately. Will be better for code readability.

**Pull request merged**